### ISSUE TITLE
Not accessible with Gnome-Orca (Screen Reader)

### ISSUE THREAD

**Issue by Nalin-x-Linux** [telegramdesktop/tdesktop#8508]
### Steps to reproduce
1. Enable orca screen reader by running command "orca"
2. Start the Telegram Desktop
3. Navigate through Telegram Desktop widgets

### Expected behaviour
The orca screen reader should announce everything in the user interface including incoming reply messages.

### Actual behaviour
Screen reader is silent. 

### Configuration
**Operating system:**
Ubuntu 20.04 LTS, Linux 5.4.0-26-generic.

**Version of Telegram Desktop:**
2.3

**Installation source (Linux Only)**
executable from the official website

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/issues/476
